pyproject.toml

```diff
@@ -75,6 +75,7 @@ tests = [
     "black",
     "flake8",
     "factory-boy",
```

Comment:
Let's also add `django-webtest` here, that way if you run `pip install -e .` you can install the dependencies locally and run `pytest tests` without tox